web/package.json

```diff
@@ -32,10 +32,10 @@
     "@rushstack/eslint-patch": "^1.5.1",
     "@types/luxon": "^3.4.2",
     "@vue/cli": "^4.5.19",
-    "@vue/cli-plugin-babel": "^4.5.19",
-    "@vue/cli-plugin-router": "^4.5.19",
-    "@vue/cli-plugin-vuex": "^4.5.19",
-    "@vue/cli-service": "^4.5.19",
+    "@vue/cli-plugin-babel": "^5.0.0",
+    "@vue/cli-plugin-router": "^5.0.0",
+    "@vue/cli-plugin-vuex": "^5.0.0",
+    "@vue/cli-service": "^5.0.0",
```

Comment:
All of these need to match.

Reply:
I'm not sure that `@vue/cli` 5 is something we can upgrade too since it might break a bunch of other stuff. I seem to recall that it did. Given that, I'd try to defer that to another PR.